pages/404html.js

```diff
@@ -0,0 +1,68 @@
+const error404 = `<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Blogium-Service / Not found route</title>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,700">
+	<style>
+		body {
+			background: #0e141a;
+			font-family: 'Open Sans', sans-serif;
+		}
+
+		.container {
```

Comment:
I think you could achieve a better result with `flexbox` instead of this `absolute + transform` combination.
I didn't test it, but I think the alternative would look something like:
```css
.container {
  display: flex;
  flex-flow: column;
  justify-content: center;
  align-items: center;
}
```